pocket_coffea/utils/plot_utils.py

```diff
@@ -395,6 +580,13 @@ def plot_data_mc_hist1D(
             else:
                 dict_mc = {d: h[d][slicing_mc] for d in samples_mc}
                 dict_mc_nominal = {d: h[d][slicing_mc_nominal] for d in samples_mc}
+                nevents = {
+                    d: round(sum(dict_mc_nominal[d].values()), 1) for d in samples_mc
```

Comment:
The number of events should not be rounded to ingeer MC events, but at least rounded to to have 1 decimal digit.

Reply:
This is what `round(sum(dict_mc_nominal[d].values()), 1)` is actually doing. When the second argument of `round()` is set to 1, the number is rounded to the first decimal digit. If the argument was instead set to 0, that would correspond to an integer rounding, but it is not the case.

Reply:
Ok thanks, I missed the argument of `round()` :)